cypress/e2e/templated-query.cy.js

```diff
@@ -56,4 +56,83 @@ describe("Templated query", () => {
     cy.contains("Finished in:");
     cy.get('.column-name').find('span').contains("Wolfgang Amadeus Mozart");
   });
+
+  it("Able to change variables after making a templated query", () => {
+    cy.visit("/");
+    cy.contains("Templated query about my favourite musicians").click();
+
+    // Fill in the query
+    cy.get('form').within(() => {
+      cy.get('#genre').click();
+    });
+    cy.get('li').contains('Baroque').click();
+
+    // Comfirm query
+    cy.get('button').contains('Query').click();
+
+    // Check that the page loaded and that we can see the correct data
+    cy.contains("Finished in:");
+    cy.get('.column-name').find('span').contains("Johann Sebastian Bach");
+
+    // Check if the button to make a new query exists and use it
+    cy.get('button').contains("Change Variables").should("exist");
+    cy.get('button').contains("Change Variables").click();
+
+    // Making sure we get the form te enter new variables
+    cy.get('form').should("exist");
```

Comment:
Better check to see if it is a form containing "genre". You never know what other forms could come later.

Reply:
updated this